src/installation/index.md

```diff
@@ -1,61 +1,167 @@
-# Rust on ESP targets
+# Setting Up a Development Environment
 
 With an understanding of the ecosystem surrounding Rust on Espressif chips, we can move on to actual development. If you are not aware of the two possible development approaches or do not understand the differences between writing `std` and `no_std` applications, please first read the [Ecosystem Overview] chapter.
 
-Let's take a moment to discuss the Rust support for the architectures of the Espressif chips in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
+Let's take a moment to discuss the Rust support for the different architectures of the Espressif chips and the different approaches in more detail. At this moment, Espressif SoCs are based on two different architectures: `RISC-V` and `Xtensa`. The support for those two architectures in the Rust programming language is very different.
 
-## Rust in RISC-V targets
+[Ecosystem Overview]: ../overview/index.md
+
+## RISC-V targets
+
+The `RISC-V` architecture has support in the mainline Rust compiler so, the setup is relatively simple. There are two ways of proceeding with the installation:
+- Using [`espup`, a tool that will be covered later]
+- Using the official Rust tools
```

Comment:
```suggestion
- Using the official Rust tools
- Using [`espup`, a tool that will be covered later]
```

I think we should encourage using official tools whenever possible, and as such it should be mentioned first IMO